src/syrupy/plugins/image/serializer.py

```diff
@@ -1,5 +1,52 @@
+from typing import Any, Set, Optional
```

Comment:
copy & pasted and then swapped "io" for "serializer". If I split this into 2 commits, maybe GitHub would pick up the diff better?

Reply:
That's annoying. I think it's just the fact that this file existed before, so it does not register it as a move/rename